### PR TITLE
Switch to Amazon Linux 2 from CoreOS

### PR DESCRIPTION
**Pull request by garymoon**

### CODE REVIEW

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -45,15 +54,10 @@ ExecStop=-/usr/bin/docker stop ${container_name}
 WantedBy=multi-user.target
 CONF
 
+systemctl daemon-reload
+
+
 systemctl enable /etc/systemd/system/conjur.service
```
Comment (jvanderhoof) [inline scripts/bootstrap.sh:60]: Can we change this to `dap.service`?

File: README.md [hunk 2]
```diff
@@ -52,11 +52,9 @@ Given a Conjur Docker image, `conjur-appliance.tar.gz`, packer runs
 1. Creates a container from the Conjur Docker image
 2. Creates and enables a service that will start this container on system boot
 
-### CoreOS
+### Amazon Linux 2
 
-CoreOS is our preferred OS. The CoreOS version and AMI we use are in [secrets.yml](secrets.yml). These are not secrets, but putting them here makes it easy to pass them into packer and test-kitchen.
-
-CoreOS uses [systemd](https://coreos.com/docs/launching-containers/launching/getting-started-with-systemd/) as its init system. The name of the service that runs the Conjur container is `conjur.service`.
+Amazon Linux 2 is our preferred OS. We use the latest version at time of build. The name of the service that runs the Conjur container is `conjur.service`.
```
Comment (jvanderhoof) [inline README.md:57]:
```suggestion
Amazon Linux 2 is our preferred OS. We use the latest version at the time of AMI build. The name of the service that runs the DAP container is `dap.service`.
```

File: scripts/bootstrap.sh (hunk 1 above)
Comment (garymoon) [reply scripts/bootstrap.sh:60]: That out of scope for this PR. We're doing the OS change only and keeping the other PR open for further changes/improvements.

Comment (jvanderhoof) [reply scripts/bootstrap.sh:60]: If changing this is super involved, I'm fine changing it at a future point.  It just feels like a good time to remove the term `Conjur` when talking about our enterprise product.

**Comment by jvanderhoof** [comment]
@garymoon, did you skip the updated folder structure intentionally?  It would be really nice to bring the AMI structure inline with our non-AMI recommendations so we can give customers a single set of setup instructions rather than maintain multiple versions.

**Comment by garymoon** [comment]
@jvanderhoof I did. The only thing in scope for this PR is changing the base OS. The old PR will remain open for further changes.

**Pull request merged**